### DISCUSSION TITLE
rabbitmq-diagnostics status error

### DISCUSSION THREAD

**Discussion by Heyz6** [rabbitmq/rabbitmq-server#5083] [Other] [answered]
docker images: rabbitmq-server:3.9.20

Cluster status of node rabbit@test-rabbitmq-0 ...
Basics

Cluster name: rabbit@test-rabbitmq-0.test-rabbitmq.default.svc.cluster.local

Disk Nodes

rabbit@test-rabbitmq-0

Running Nodes

rabbit@test-rabbitmq-0

Versions

rabbit@test-rabbitmq-0: RabbitMQ 3.9.20 on Erlang 24.3.4.1

Maintenance status

Node: rabbit@test-rabbitmq-0, status: not under maintenance

Alarms

(none)

Network Partitions

(none)

Listeners

Node: rabbit@test-rabbitmq-0, interface: [::], port: 15672, protocol: http, purpose: HTTP API
Node: rabbit@test-rabbitmq-0, interface: [::], port: 25672, protocol: clustering, purpose: inter-node and CLI tool communication
Node: rabbit@test-rabbitmq-0, interface: [::], port: 5672, protocol: amqp, purpose: AMQP 0-9-1 and AMQP 1.0

Feature flags

Flag: drop_unroutable_metric, state: enabled
Flag: empty_basic_get_metric, state: enabled
Flag: implicit_default_bindings, state: enabled
Flag: maintenance_mode_status, state: enabled
Flag: quorum_queue, state: enabled
Flag: stream_queue, state: enabled
Flag: user_limits, state: enabled
Flag: virtual_host_metadata, state: enabled

--------------------------
cat /etc/rabbitmq/rabbitmq.conf
loopback_users.guest = false
vm_memory_high_watermark.absolute = 0.6
disk_free_limit.absolute = 3GB

root@test-rabbitmq-0:/# rabbitmq-diagnostics ping
Will ping rabbit@test-rabbitmq-0. This only checks if the OS process is running and registered with epmd. Timeout: 60000 ms.
Ping succeeded
root@test-rabbitmq-0:/# rabbitmq-diagnostics status
Status of node rabbit@test-rabbitmq-0 ...
Stack trace:

** (FunctionClauseError) no function clause matching in RabbitMQ.CLI.InformationUnit.convert/2
    (rabbitmqctl 3.8.0-dev) lib/rabbitmq/cli/information_unit.ex:33: RabbitMQ.CLI.InformationUnit.convert({:error, :parse_error}, "gb")
    (rabbitmqctl 3.8.0-dev) lib/rabbitmq/cli/ctl/commands/status_command.ex:137: RabbitMQ.CLI.Ctl.Commands.StatusCommand.output/2
    (rabbitmqctl 3.8.0-dev) lib/rabbitmqctl.ex:153: RabbitMQCtl.maybe_run_command/3
    (rabbitmqctl 3.8.0-dev) lib/rabbitmqctl.ex:121: anonymous fn/5 in RabbitMQCtl.do_exec_parsed_command/5
    (rabbitmqctl 3.8.0-dev) lib/rabbitmqctl.ex:584: RabbitMQCtl.maybe_with_distribution/3
    (rabbitmqctl 3.8.0-dev) lib/rabbitmqctl.ex:93: RabbitMQCtl.exec_command/2
    (rabbitmqctl 3.8.0-dev) lib/rabbitmqctl.ex:31: RabbitMQCtl.main/1
    (elixir 1.13.4) lib/kernel/cli.ex:126: anonymous fn/3 in Kernel.CLI.exec_fun/2

Error:
:function_clause

**Comment by michaelklishin** [comment]
Something about your config trips up the limit value parser. We cannot suggest what it may be without the `rabbitmq.conf` contents.

**Comment by michaelklishin** [comment, marked as answer]
We now know what it was: an unsupported information unit in `rabbitmq.conf`: https://github.com/rabbitmq/rabbitmq-server/discussions/10308?converting=1.
